src/sentry/static/sentry/app/components/events/interfaces/spans/index.tsx

```diff
@@ -66,4 +174,43 @@ const StyledSearchBar = styled(SearchBar)`
   margin-bottom: ${space(1)};
 `;
 
-export default SpansInterface;
+const AlertMessageContainer = styled('div')`
+  margin-bottom: ${space(1)};
+`;
+
+function filterSpansWithErrors(
+  parsedTrace: ParsedTraceType,
+  tableData: TableData | null | undefined
+): TableData | null | undefined {
+  if (!tableData) {
+    return tableData;
```

Comment:
Couldn't we narrow the return types by doing `return null` here?

Reply:
done in https://github.com/getsentry/sentry/pull/18025/commits/0fbb16ee6f898acb3112437c386c319b266f0fa3